### PR TITLE
Add JSON validator for .md files inside the content directory

### PR DESCRIPTION
**Pull request by rgeniesse**
## Description
Validates JSON within tags `{{< highlight json >}} {{< /highlight >}}` using Python's json library.

## Motivation and Context
Ensures future JSON blocks are valid inside the `content` directory.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,13 +1,25 @@
...
-      - python-pygments
+      addons:
+        apt:
+          packages:
+            - python-pygments
 
-before_install:
-  - export PATH="$TRAVIS_BUILD_DIR/bin:$PATH"
+      before_install:
+        - export PATH="$TRAVIS_BUILD_DIR/bin:$PATH"
```
Comment (apaskulin) [inline .travis.yml:13]: Are these spacing changes intentional?

Comment (rgeniesse) [reply .travis.yml:13]: Yes. YAML depends on spacing to dictate what an object lives under. So as I added a second language here, I needed to move these over they would be under `matrix` and `include`. 

The spacing that doesn't matter are the new lines between objects, but I just followed what was here to be consistent.

Comment (apaskulin) [reply .travis.yml:13]: Perfect. Thanks!

**Review by apaskulin — APPROVED** [review]
This is awesome! So happy to have this added to the project. I wasn't able to test this, but I'm +1 to merging and iterating. :tada: ⭐️

**Pull request merged**